docs/index.rst

```diff
@@ -13,17 +13,34 @@ research ideas and products based on these models. This toolkit offers four main
 
 
 This toolkit assumes that users have basic knowledge about deep learning and
-NLP. Otherwise, please refer to an introduction course such as 
+NLP. Otherwise, please refer to an introduction course such as
 `Deep Learning---The Straight Dope <http://gluon.mxnet.io/>`_ or
 `Stanford CS224n <http://web.stanford.edu/class/cs224n/>`_.
 
+The toolkit supports the following NLP tasks:
+
+1. :doc:`model_zoo/word_embeddings/index`
+2. :doc:`model_zoo/language_model/index`
+3. :doc:`model_zoo/machine_translation/index`
+4. :doc:`model_zoo/text_classification/index`
+5. :doc:`model_zoo/sentiment_analysis/index`
+6. :doc:`model_zoo/text_generation/index`
+
 .. note::
 
-   This project is at an early stage and is under active development.
-   Please expect that it will be updated frequently. Contributions are welcome.
+   We can be found on `Github <https://github.com/dmlc/gluon-nlp>`_. This project is at an early stage
+   and is under active development, so please expect frequent updates.
+   Check out our `contribution guide <http://gluon-nlp.mxnet.io/how_to/contribute.html>`_
+   to see how you can help.
+
+
+.. hint::
+
+   You can find our the doc for our master development branch `here <http://gluon-nlp.mxnet.io/master/index.html>`_.
+
 
 Installation
-------------------
+------------
 
 GluonNLP relies on the recent version of MXNet. The easiest way to install MXNet
 is through `pip <https://pip.pypa.io/en/stable/installing/>`_. The following
```

Comment:
```
pip install --pre --upgrade mxnet
```
->
```
mxnet 1.3.0
```

Reply:
The next release may again include code relying on the --pre version, so perhaps leaving it the same would be less confusing?